xla/python/xla_client_test.py

```diff
@@ -3169,17 +3169,12 @@ def testCompileOptionsSerialization(self):
       options.compile_portable_executable = True
       executable_build_options.num_replicas = 3
       executable_build_options.num_partitions = 2
-      executable_build_options.debug_options.xla_cpu_enable_fast_math = True
-      executable_build_options.debug_options.xla_test_all_input_layouts = True
-      executable_build_options.debug_options.xla_gpu_kernel_cache_file = (
-          "/foo/bar"
-      )
-      executable_build_options.debug_options.xla_gpu_enable_llvm_module_compilation_parallelism = (
-          True
-      )
-      executable_build_options.debug_options.xla_gpu_per_fusion_autotune_cache_dir = (
-          "/bar/foo/"
-      )
+      deb_opt = executable_build_options.debug_options
+      deb_opt.xla_cpu_enable_fast_math = True
+      deb_opt.xla_test_all_input_layouts = True
+      deb_opt.xla_gpu_kernel_cache_file = ("/foo/bar")
+      deb_opt.xla_gpu_enable_llvm_module_compilation_parallelism = (True)
+      deb_opt.xla_gpu_per_fusion_autotune_cache_dir = ("/bar/foo/")
```

Comment:
No need for parenthesis on these lines. I assume they were initially added to make the lines shorter (but not short enough).

Reply:
fixed